augur/clades.py

```diff
@@ -147,16 +146,17 @@ def assign_clades(clade_designations, all_muts, tree, ref=None):
         sorted_nodes = sorted(node_counts, key=lambda x: x.leaf_count, reverse=True)
         if len(sorted_nodes) > 0:
             target_node = sorted_nodes[0]
-            clade_membership[target_node.name] = {'clade_annotation': clade_name, 'clade_membership': clade_name}
+            basal_clade_nodes[target_node.name] = clade_name
+            clade_membership[target_node.name] = clade_name # basal nodes are members of the clade
 
-    # third pass to propagate 'clade_membership'
+    # propagate 'clade_membership' to children nodes
```

Comment:
I'm still a bit unsure about all this.

Nodes not part of clades ("undefined") aren't part of the output of `augur clades`, so explicitly annotating the root node as such would be strange. 

In the nCoV workflow the root node is annotated (both node + branch) as clade 19A, so `augur clades` does have this functionality. I _think_ it needs the sequence annotated in the node-data JSON inputs, it can't infer this from the observed mutations, which is something we should test / document.